src/cmd/build.js

```diff
@@ -94,13 +94,24 @@ export function safeFileName(name: string): string {
  */
 export class FileFilter {
   filesToIgnore: Array<string>;
-
-  constructor({filesToIgnore}: Object = {}) {
+  dirToIgnore : Array<string>;
+  
+  constructor(artifactsDir,{filesToIgnore,dirToIgnore}: Object = {}) {
```

Comment:
Accepting `artifactsDir` as an argument is not necessary. The FileFilter doesn't need to know about the artifacts directory, it just needs to know which file paths to ignore.

Instead of calling the FileFilter like this:

    const filter = new FileFilter(artifactsDir);

you could call it like this:

    const filter = new FileFilter({filePathsToIgnore: [path.resolve(artifactsDir)]});

This makes the FileFilter more flexible; you could pass in as many file paths to ignore as necessary.